src/AdoNetCore.AseClient/Enum/BackupServerErrorMajor.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AdoNetCore.AseClient.Enum
```

Comment:
I don't think this is necessary either, since it also relates to parsing the messages.